dask/array/tests/test_array_core.py

```diff
@@ -414,6 +431,7 @@ def test_norm():
     assert b.vnorm(ord=4, axis=0, keepdims=True).ndim == b.ndim
```

Comment:
I suspect that a fair amount of this actually does work in numpy < 1.8 and it's just a few functions that fail.

We don't necessarily need dask.array to pass a numpy 1.6 test suite.  We just need it to be importable.